### ISSUE TITLE
Races

### ISSUE THREAD

**Issue by luluwaza** [IGLADI/Ethereal-Hyperspace-Battleships#3]
Players need to pick a race during the tutorial phase of the game, they have 3 classes to choose between:
- Martian: longer cooldown on travel commands, more energy in their ship
- Dwarf: Gets more resources out of mining see `/mine`, sells resources for higher price at shops
- Droid: better healer, repairs damaged ships faster, has bonus when refueling other players

**Comment by ThomasKnoops** [comment]
This seems already implemented inside the database branch? Or am I wrong?

**Comment by LordTlasT** [comment]
> This seems already implemented inside the database branch? Or am I wrong?

No, we store their classes and that's it.  They are not even a property of the player class.

**Comment by LordTlasT** [comment]
waiting for #75.

**Comment by ThomasKnoops** [comment]
https://github.com/IGLADI/Ethereal-Hyperspace-Battleships/pull/75 is done... this is good to go!
